include/boost/unordered/unordered_map.hpp

```diff
@@ -1773,6 +1794,16 @@ namespace boost {
 
 #if !defined(BOOST_NO_CXX11_HDR_INITIALIZER_LIST)
 
+    template <class K, class T, class H, class P, class A>
+    unordered_map<K, T, H, P, A>::unordered_map(
+      std::initializer_list<value_type> list, const allocator_type& a)
+        : table_(boost::unordered::detail::initial_size(
+                   list.begin(), list.end(), detail::default_bucket_count),
```

Comment:
... but this passes detail::default_bucket_count. Is this intended?

Reply:
The other two places both use detail::default_bucket_count, so probably not.

Reply:
Good catch, we should be using `default_bucket_count` everywhere.